packages/components/src/Lettera/Models.purs

```diff
@@ -633,3 +639,105 @@ editorialIdToUuid editorialId =
 url_namespace :: UUID.UUID
 url_namespace = unsafePartial $ fromJust $ UUID.parseUUID "6ba7b811-9dad-11d1-80b4-00c04fd430c8"
 
+encodeRequestHeaders :: Array AX.RequestHeader -> Maybe Json
```

Comment:
I think `Lettera.Models` is too busy already and nothing in here uses these so it's not justified by dependencies either. The focus of this module is diluted if it gets Affjax imports to it. If you're moving these from `Lettera` make it a new module. `Lettera.Requests` or if you have a better idea.